code/src/components/ThoughtsInput.js

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+
+const ThoughtsInput = ({
+  onFormSubmit,
+  setCounterValue,
+  newThought,
+  setNewThought,
+}) => {
+  const RefreshButton = () => {
```

Comment:
Nice little feature that you added, maybe look into adding a hover cursor for this button as well?